tests/integration/components/project-header-test.js

```diff
@@ -0,0 +1,26 @@
+import { moduleForComponent, test } from 'ember-qunit';
+import hbs from 'htmlbars-inline-precompile';
+import startMirage from '../../helpers/setup-mirage-for-integration';
+
+moduleForComponent('project-header', 'Integration | Component | project header', {
+  integration: true,
+  setup() {
+    startMirage(this.container);
+  },
+  afterEach() {
+    server.shutdown();
+  }
+});
+
+test('it displays the right title, icon, description', function(assert) {
+  assert.expect(3);
+
+  let project = server.create('project');
```

Comment:
What are your thoughts on not using mirage for this test?  This implies that the component relies on the store, when really it just takes in a model with a `title`, `description` and `iconThumbUrl`.

We could update this to be:
`this.set('project', { title: 'foo', description: 'bar', iconThumbUrl: 'image/thing.jpg' })`

Then we don't have to worry about starting mirage for this test and it enforces that this component is stateless (as it shouldn't care what that model looks like or how it is served).